driverthroughput/README.md

```diff
@@ -0,0 +1,253 @@
+    --db_port=DB_PORT
+        Default: 6780
+        port to use for the database
+    --net_ifname=NET_IFNAME
+        Default: 'ipogif0'
+        network interface to use i.e. "ib0" for infiniband or "ipogif0" aries networks
+    --clients_per_node=CLIENTS_PER_NODE
+        Default: [48]
+        client tasks per compute node for the synthetic scaling producer app
+    --client_nodes=CLIENT_NODES
+        Default: [4,8,16,32,64,128]
+        number of compute nodes to use for the synthetic scaling producer app
+    --iterations=ITERATIONS
+        Default: 100
+        number of put/get loops run by the applications
+    --tensor_bytes=TENSOR_BYTES
+        Default: [1024, 8192, 16384, 32769, 65538, 131076, 262152, 524304, 10...
+        list of tensor sizes in bytes
+    --languages=LANGUAGES
+        Default: ['cpp']
+        which language to use for the tester "cpp" or "fortran"
+    --wall_time=WALL_TIME
+        Default: '05:00:00'
+        allotted time for database launcher to run
+    --plot=PLOT
+        Default: 'database_nodes'
+        flag to plot against in process results
+```
+
+The standard throughput tests will spin up a database for each iteration in the
+battery of tests chosen by the user. There are multiple ways to run this.
+
+1. Everything in the same interactive (or batch file) without caring about placement
+```bash
+# alloc must contain at least 60 (max client_nodes) + 32 nodes (max db_nodes)
+python driver.py throughput_standard --client_nodes=[60] \
+                                    --clients_per_node=[48] \
+                                    --db_nodes=[32] \
+                                    --db_cpus=[32] --net_ifname="ipogif0" \
+                                    --run_db_as_batch=False
+```
+
+This option is recommended as it's easy to launch in interactive allocations and
+as a batch submission, but if you need to specify separate hosts for the database
+you can look into the following two methods.
+
+A batch submission for this first option would look like the following for Slurm
+based systems.
+
+```bash
+#!/bin/bash
+
+#SBATCH -N 92
+#SBATCH --exclusive
+#SBATCH -t 10:00:00
+#SBATCH -C SK48
+#SBATCH --oversubscribe
+
+cd ..
+python driver.py throughput_standard --client_nodes=[60] \
+                                    --clients_per_node=[48] \
+                                    --db_nodes=[32] \
+                                    --db_cpus=[32] --net_ifname="ipogif0" \
+                                    --run_db_as_batch=False
+```
+
+2. Same as 1, but specify hosts for the database
+```bash
+# alloc must contain at least 60 (max client_nodes) + 32 nodes (max db_nodes)
+# db nodes must be fixed if hostlist is specified
+python driver.py throughput_standard --client_nodes=[60] \
+                                    --clients_per_node=[48] \
+                                    --db_nodes=[32] \
+                                    --db_cpus=[32] --net_ifname="ipogif0" \
```

Comment:
here and elsewhere, should be put in a note that this interface name may not be standard and should be replaced with whatever is appropriate on the target system?

Reply:
I don't see that this is addressed